src/WebJobs.Script.WebHost/Middleware/SystemTraceMiddleware.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.WebJobs.Script.Extensions;
+using Microsoft.Azure.WebJobs.Script.WebHost.Security.Authentication;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Middleware
+{
+    internal class SystemTraceMiddleware
+    {
+        private readonly ILogger _logger;
+        private readonly RequestDelegate _next;
+
+        public SystemTraceMiddleware(RequestDelegate next, ILogger<SystemTraceMiddleware> logger)
+        {
+            _logger = logger;
+            _next = next;
+        }
+
+        public async Task Invoke(HttpContext context)
+        {
+            SetRequestId(context.Request);
+
+            var request = context.Request;
+            var details = new JObject
+            {
+                { "requestId", request.GetRequestId() },
+                { "method", request.Method.ToString() },
+                { "uri", request.Path.ToString() }
+            };
+            var logData = new Dictionary<string, object>
+            {
+                [ScriptConstants.LogPropertyActivityIdKey] = request.GetRequestId()
+            };
+            _logger.Log(LogLevel.Information, 0, logData, null, (s, e) => $"Executing HTTP request: {details}");
+
+            await _next.Invoke(context);
+
+            details["identities"] = GetIdentities(context);
+            details["status"] = context.Response.StatusCode.ToString();
+            _logger.Log(LogLevel.Information, 0, logData, null, (s, e) => $"Executed HTTP request: {details}");
```

Comment:
Did we log these as Information in v1? Not sure what the standard is -- it would add 2 traces to every Http function by default.

Reply:
Yes we logged this in v1 - see the source link I shared.